BMW/bmw/page-2.html

```diff
@@ -58,19 +58,26 @@ <h2 style="color: white">
             ></font
           >
         </p>
-        <ul style="color:white;">
+        <ul style="color: white">
           <li>
-            <font size="5"> на базовой версии он генерирует 480 лошадиных сил при 6250 об/минуту
-            и 550 Нм крутящего момента при 2650-6130 об/минуту;</font>
+            <font size="5">
+              на базовой версии он генерирует 480 лошадиных сил при 6250
+              об/минуту и 550 Нм крутящего момента при 2650-6130
```

Comment:
You can write tag in one row and all text in another one, like this
```suggestion
<font size="5">
  на базовой версии он генерирует 480 лошадиных сил при 6250 об/минуту и 550 Нм крутящего момента при 2650-6130об/минуту;
</font>
```